sansa-ml-spark/src/main/scala/net/sansa_stack/ml/spark/entity_resolution/Commons.scala

```diff
@@ -88,7 +88,7 @@ abstract class Commons(val spark: SparkSession, val sourceData1: RDD[Triple], va
           val value = pred + ":" + obj // predicate and object are seperated by ':'
           (key, value)
         } else {
-          val obj = f.getObject.getLiteralValue.toString()
+          val obj = f.getObject.getLiteral.toString().split(Array('^', '@')).head.trim()
```

Comment:
Would it be better if we get the [lexical form](https://jena.apache.org/documentation/javadoc/jena/org/apache/jena/rdf/model/Literal.html#getLexicalForm--) of the literal? which will remove everything i.e. language tag, type, etc? Or is this method meant for something else?

Reply:
Yes, It does the same.